oop/src/store/modules/form.js

```diff
@@ -38,6 +39,7 @@ export default {
       phone: null,
       moveFromBCDate: null,
       arriveDestinationDate: null,
+      newAddressIsKnown: null,
```

Comment:
Since it's a boolean, can you start the variable name with "is" (`isNewAddressKnown` instead of `newAddressIsKnown`)? Same thing in all places this is used in the PR.